keras_hub/src/models/gemma3/gemma3_image_converter.py

```diff
@@ -6,3 +6,9 @@
 @keras_hub_export("keras_hub.layers.Gemma3ImageConverter")
 class Gemma3ImageConverter(ImageConverter):
     backbone_cls = Gemma3Backbone
+
+    def __init__(self, **kwargs):
+        # Always do image preprocessing in float32
+        kwargs.pop("dtype", None)
+        dtype = "float32"
```

Comment:
why is this btw? won't the images get converted to the compute dtype later?

Reply:
I want to do standardisation, normalisation in `float32` because these ops are sensitive to precision. What do you think?